go-runtime/schema/extractor.go

```diff
@@ -21,7 +21,8 @@ func Extract(moduleDir string) (analyzers.ExtractResult, error) {
 		RunDespiteLoadErrors: true,
 		Patterns:             []string{"./..."},
 	}
-	results, diagnostics, err := checker.Run(cConfig, append(analyzers.Extractors, analyzers.Finalizer)...)
+	as := []*analysis.Analyzer{analyzers.Initializer, analyzers.Finalizer}
```

Comment:
Not related to this line, just a general comment: did you happen to check if we need to add any more test coverage? I can't recall if we're missing any significant branch cases for the new approach.